gesos-image-build.pipeline

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/env groovy
+
+@Library(['PPCmanifest','security-ci-commons-shared-lib']) _
```

Comment:
None of these libraries are needed for this pipeline.  
PPCmanifest ia a lib used in the past to publish UAA war to Bintray for GovCloud; now obsolete.
security-ci-commons-shared-lib is a lib to pick an agent for the pipeline.  Since you are explicitly set the agent to  "openjdk:8", lib is not needed.

Reply:
Removed unnecessary libs.